app/pages/sections/additional-services/order-item/flat/patient/ui-tests/general.ui.test.js

```diff
@@ -0,0 +1,258 @@
+    .expect(expandableSection.find('details[open]').exists).ok()
+    .expect(await extractInnerText(expandableSection.find('.nhsuk-details__text')))
+    .eql(content.questions.quantity.expandableSection.innerComponent);
+});
+
+test('should render the price table headings', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const priceTable = Selector('div[data-test-id="price-table"]');
+  const priceColumnHeading = priceTable.find('[data-test-id="column-heading-0"]');
+  const unitColumnHeading = priceTable.find('[data-test-id="column-heading-1"]');
+
+  await t
+    .expect(priceTable.exists).ok()
+    .expect(priceColumnHeading.exists).ok()
+    .expect(await extractInnerText(priceColumnHeading)).eql(content.addPriceTable.columnInfo[0].data)
+    .expect(unitColumnHeading.exists).ok()
+    .expect(await extractInnerText(unitColumnHeading)).eql(content.addPriceTable.columnInfo[1].data);
+});
+
+test('should render the price table content', async (t) => {
+  await pageSetup();
+  await t.navigateTo(pageUrl);
+
+  const table = Selector('div[data-test-id="price-table"]');
+  const row = table.find('[data-test-id="table-row-0"]');
+  const priceInput = row.find('[data-test-id="question-price"] input');
+  const expandableSection = row.find('[data-test-id="view-section-input-id"]');
+  const orderUnit = row.find('div[data-test-id="unit-of-order"]');
+
+  await t
+    .expect(row.exists).ok()
+    .expect(priceInput.exists).ok()
+    .expect(priceInput.getAttribute('value')).eql(selectedPrice.price.toString())
+    .expect(expandableSection.exists).ok()
+    .expect(await extractInnerText(expandableSection)).eql(content.addPriceTable.cellInfo.price.expandableSection.title)
+    .expect(expandableSection.find('details[open]').exists).notOk()
+    .click(expandableSection.find('summary'))
+    .expect(expandableSection.find('details[open]').exists).ok()
+    .expect(await extractInnerText(expandableSection.find('.nhsuk-details__text')))
+    .eql(content.addPriceTable.cellInfo.price.expandableSection.innerComponent)
+    .expect(orderUnit.exists).ok()
+    .expect(await extractInnerText(orderUnit)).eql(selectedPrice.itemUnit.description);
+});
+
+test('should render select quantity field as errors with error message when no quantity entered causing validation error', async (t) => {
+  await pageSetup(true, true);
+  await t.navigateTo(pageUrl);
+
+  const orderItemPage = Selector('[data-test-id="order-item-page"]');
+  const saveButton = Selector('[data-test-id="save-button"] button');
+  const quantityField = orderItemPage.find('[data-test-id="question-quantity"]');
+  const button = Selector('[data-test-id="save-button"] button');
+
+  await t
+    .expect(button.exists).ok()
+    .expect(quantityField.find('[data-test-id="text-field-input-error"]').exists).notOk()
+    .click(saveButton);
+
+  await t
+    .expect(quantityField.find('[data-test-id="text-field-input-error"]').exists).ok()
```

Comment:
could this be defined as a const?

Reply:
yes that is nicer. Will update and the others too to keep consistent.